os_helper.bash

```diff
@@ -139,13 +139,11 @@ tRHSubscribeAttach() {
 tRHEnableEPEL() {
   tIsRHEL || skip "Not required"
   if tIsRHEL 7; then
```

Comment:
I noticed that EL6 used rpm where EL7 used yum so I thought to leave that in. Any reason they use different commands?